### DISCUSSION TITLE
Ever thought of adding a couple vpncloud videos on YouTube ?

### DISCUSSION THREAD

**Discussion by bmullan** [dswd/vpncloud#154] [Ideas]
Short Youtube videos can be pretty easy to create.  One of the tools I've used is SimpleScreenRecorder which I learned in less than 10 minutes.

Youtube videos can make things so much more understandable for End-Users & New-Users for basic things like  
- key features
- new install
- configuration
- basic troubleshooting

and each video can point to our vpncloud github repository.

**Comment by dswd** [comment]
I once recorded screencasts for a different project to show some graphical user interface. However back then I noticed quite some downsides to video tutorials:
1. Videos are not searchable. Users can only find videos by their title and description but not by the content (i.e. what you say and show in the video).
2. Videos are hard to edit. When some things change about your software you need to adapt the tutorials. With videos this often means record them again. Also it is hard to edit out some information you don't want be in the video (like typos or information that would distract the viewer).

Since VpnCloud is a command line application, I prefer text descriptions which are easier to edit and also searchable. Additionally, users can copy-paste commands directly from the website to their terminal.

I want to put some feature explanations as well as tutorials on the VpnCloud website at https://vpncloud.ddswd.de/docs/
If at some point a visual live tutorial would be better, I would prefer something like https://asciinema.org/.

Have you seen the beginner tutorial at https://vpncloud.ddswd.de/docs/tutorials/basic/ ? What kind of information do you think should be added there?
